modules/programs/nushell.nix

```diff
@@ -39,7 +39,7 @@ let
       };
     });
 in {
-  meta.maintainers = [ maintainers.Philipp-M ];
+  meta.maintainers = [ maintainers.Philipp-M maintainers.joaquintrinanes ];
```

Comment:
I think you also need to add yourself in the `maintainers.nix` file.

(Btw. I think you can name yourself as first maintainer, I'm not really effectively maintaining nushell anyways, you seem to be more into nushells guts)

Reply:
It shouldn't be necessary to add yourself to the HM maintainers file if you are already in the Nixpkgs' one.